flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/expressions/aggregations.scala

```diff
@@ -57,7 +58,10 @@ case class Sum(child: PlannerExpression) extends Aggregation {
   override private[flink] def children: Seq[PlannerExpression] = Seq(child)
   override def toString = s"sum($child)"
 
-  override private[flink] def resultType = child.resultType
+  override private[flink] def resultType = {
```

Comment:
> [FLINK-13107][table-planner-blink] Fix bug when infer resultType of some PlannerExpression.
[FLINK-13107][table-planner-blink] Derive sum, avg, div return type using behavior of blink ?

Reply:
I think we should use behavior of blink.

Reply:
Yeah, The change is very good, I think the commit message should be more clear.